server/app/controllers/admin/AdminProgramBlockPredicatesController.java

```diff
@@ -426,4 +431,14 @@ public Result updateEligibilityMessage(Request request, long programId, long blo
     return redirect(indexUrl)
         .flashing(toastType.toString().toLowerCase(Locale.getDefault()), toastMessage);
   }
+
+  /** HTMX partial that renders a card for editing a condition within a predicate. */
+  @Secure(authorizers = Authorizers.Labels.CIVIFORM_ADMIN)
+  public Result hxEditCondition(Request request) {
+    DynamicForm formData = formFactory.form().bindFromRequest(request);
```

Comment:
(blocking) Before anything else check that your feature flag is enabled and return `notFound` if it is not on.